docs/versions/unversioned/sdk/background-fetch.md

```diff
@@ -17,13 +16,16 @@ Returns a promise resolving to one of these values:
 -   `BackgroundFetch.Status.Denied` - The user explicitly disabled background behavior for this app or for the whole system.
 -   `BackgroundFetch.Status.Available` - Background updates are available for the app.
 
-### `BackgroundFetch.registerTaskAsync(taskName)`
+### `BackgroundFetch.registerTaskAsync(taskName, options)`
 
-Registers background fetch task with given name. Registered tasks are saved in persistent storage and restored once the app is initialized. Remember to also call [BackgroundFetch.setMinimumIntervalAsync](#backgroundfetchsetminimumintervalasyncminimuminterval) as fetch operations will not be occurring with its default value.
+Registers background fetch task with given name. Registered tasks are saved in persistent storage and restored once the app is initialized.
+On iOS, it is also required to call [BackgroundFetch.setMinimumIntervalAsync](#backgroundfetchsetminimumintervalasyncminimuminterval) as fetch operations will not be occurring with its default value.
 
 #### Arguments
 
--   **taskName (_string_)** -- Name of the task to register. The task needs to be defined first - see [TaskManager.defineTask](../task-manager/#taskmanagerdefinetasktaskname-task) for more details.
+-   **taskName (_string_)** -- Name of the task to register. The task needs to be defined first - see [TaskManager.defineTask](../task-manager#taskmanagerdefinetasktaskname-task) for more details.
+-   **options (_object_)** -- An object of options:
+    -   **interval (_number_)** -- Inexact interval in seconds between subsequent repeats of the background fetch alarm. The final interval may differ from the specified one to minimize wakeups and battery usage. Defaults to **15 minutes**. (**Android only**, see [BackgroundFetch.setMinimumIntervalAsync](#backgroundfetchsetminimumintervalasyncminimuminterval) for iOS alternative)
```

Comment:
Maybe on iOS we could call `BackgroundFetch.setMinimumIntervalAsync` behind the scenes when this option is provided? 🤔